README.md

```diff
@@ -884,6 +888,12 @@ ANSWERS.addComponent('Pagination', {
   },
   // Function invoked when a user clicks to change pages. By default, scrolls the user to the top of the page.
   onPaginate: (newPageNumber, oldPageNumber, totalPages) => {},
+  // DEPRECATED with showFirstAndLastButton.
+  // Display a double arrow allowing users to jump to the first page of results. Defaults to showFirstAndLastButton.
+  showFirst: true,
+  // DEPRECATED with showFirstAndLastButton.
```

Comment:
I read this as "both showFirstAndLastButton and showLast are deprecated". Could we try something like `DEPRECATED, use x instead`

Reply:
yeah that's a lot better